src/audio/load.c

```diff
@@ -103,7 +106,7 @@ char loadAudioString38[] = "---------------------------------------\n";
 /**
  * Performs an immediate DMA copy
  */
-void audio_dma_copy_immediate(uintptr_t devAddr, void *vAddr, size_t nbytes) {
+void audio_dma_copy_immediate(u8 *devAddr, void *vAddr, size_t nbytes) {
```

Comment:
Are you sure this shuold be u8 * and not uintptr_t?

uintptr_t should be used when doing pointer math. If there's no ptr math then it's probably fine.

Reply:
I make it to have less errors possible and because all data gives as this argument are u8*

Reply:
Took a look. Yea, this is fine.